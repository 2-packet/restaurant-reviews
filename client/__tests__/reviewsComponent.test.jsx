import React from 'react';
import renderer from 'react-test-renderer';
import Reviews from '../src/components/reviewsComponent.jsx';

it('renders the reviews top-level component', () => {
  const tree = renderer
    .create(<Reviews restaurantId={1} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
