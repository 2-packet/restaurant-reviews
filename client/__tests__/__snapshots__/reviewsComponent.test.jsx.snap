// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the reviews top-level component 1`] = `
<div
  id="reviews"
>
  <div>
    <h3>
      Sort by
    </h3>
  </div>
  <div>
    <h1>
      reviews render here
    </h1>
  </div>
</div>
`;
